src/auth-service/bin/kafka-consumer.js

```diff
@@ -1,26 +1,114 @@
 const { Kafka } = require("kafkajs");
+const constants = require("@config/constants");
 const log4js = require("log4js");
-const logger = log4js.getLogger("Kafka Consumer");
+const logger = log4js.getLogger(`${constants.ENVIRONMENT} -- Kafka Consumer`);
+const { logText, logObject } = require("@utils/log");
+const mailer = require("@utils/mailer");
+const emailTemplates = require("@utils/email.templates");
+const Joi = require("joi");
+const { jsonrepair } = require("jsonrepair");
+
+const userSchema = Joi.object({
```

Comment:
the kafka consumer logic is now in its own separate file to make unit testing easier. @BenjaminSsempala